en/developer_guide/content_add_special_collection_field.rst

```diff
@@ -0,0 +1,132 @@
+Add a special collection field to a Content Attribute
+=====================================================
+
+Context
+-------
+
+When working with the Content and ContentType, you might have the need to save a specific field such as a collection.
```

Comment:
I think you should present the doc by talking of the listener issue before talking of the collection example.